includes/wikia/VariablesBase.php

```diff
@@ -1277,11 +1277,11 @@
 $wgCreateEnglishWikisOnFandomCom = false;
 
 /**
- * Wiki description enter by the user on the CNW page. Removed after wiki is created
- * @see extensions/wikia/CreateNewWiki/maintenance/setMainPageContent.php
- * @var string $wgWikiDescription
+ * The datacenter that hosts the starter dumps for CreateNewWiki
+ * @see \Wikia\CreateNewWiki\Starters
+ * @var string $wgCreateWikiStarterDumpsDatacenter
  */
-$wgWikiDescription = '';
+$wgCreateWikiStarterDumpsDatacenter = WIKIA_DC_SJC;
```

Comment:
I'd put this to `VariablesExpansions.php`. This file should change the state in the same way, always. Now it depends on a constant defined elsewhere.